jobbergate-agent/jobbergate_agent/jobbergate/submit.py

```diff
@@ -184,6 +190,24 @@ def run(self, *args, **kwargs):
         return super().run(*args, **kwargs)
 
 
+def validate_submit_dir(submit_dir: Path, subprocess_handler: SubprocessAsUserHandler) -> None:
```

Comment:
We should have unit tests for this function specifically.

Reply:
Addressed by [04257e7](https://github.com/omnivector-solutions/jobbergate/pull/713/commits/04257e7c66b728095aa15bbeac2acb0c16b5b88e).